### PR TITLE
yamllint formatting cleanup for ocp-workload-parksmap-demo

### PR DESCRIPTION
**Pull request by ritesh97-rh**
##### SUMMARY
For the specified roles:
- Create a yamllint file to enforce formatting rules
- Make changes to existing code base to adhere to formatting requirements
- yamllint errors will be displayed during formatting checks in future development

##### ISSUE TYPE
- Code Cleanup Request

##### COMPONENT NAME
ocp-workload-parksmap-demo

### CODE REVIEW

File: ansible/roles/ocp-workload-parksmap-demo/files/workshop-parksmap.yaml [hunk 1]
```diff
@@ -71,7 +72,7 @@ items:
           ports:
           - containerPort: 8080
             protocol: TCP
-          resources: {}
+       #  resources: {}
```
Comment (stencell) [inline ansible/roles/ocp-workload-parksmap-demo/files/workshop-parksmap.yaml:75]: Why was this commented out compared to original?

**Review by stencell — CHANGES_REQUESTED** [review]
see comment to address.

File: ansible/roles/ocp-workload-parksmap-demo/files/workshop-parksmap.yaml (hunk 1 above)
Comment (ritesh97-rh) [reply ansible/roles/ocp-workload-parksmap-demo/files/workshop-parksmap.yaml:75]: `resources` appears twice as as key in the same scope (open the whole file; don't think its visible in the snippet). The second one has a value, so I commented this one out .. should verify before merging.

Comment (stencell) [reply ansible/roles/ocp-workload-parksmap-demo/files/workshop-parksmap.yaml:75]: These are different. Even though it is empty in both cases and shouldn't make a difference, we also should not be changing something like this - that is up to the workload owner.

`.spec.strategy.resources`
vs
`.spec.template.spec.containers.resources`

Comment (ritesh97-rh) [reply ansible/roles/ocp-workload-parksmap-demo/files/workshop-parksmap.yaml:75]: Are you sure? There's another `resources: ` on line 77 that has limits and requests in it. I think that's the one yamllint says is conflicting.

Comment (stencell) [reply ansible/roles/ocp-workload-parksmap-demo/files/workshop-parksmap.yaml:75]: I beg your pardon...you are absolutely right, I missed that one.

In this case, can you just the delete the one with the empty dict? That should resolve this.

**Review by stencell — APPROVED** [review]
lgtm

**Pull request merged**